### PR TITLE
Freeze on anything using .proc/ until 515 is merged

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
This PR is a notice that there's a freeze on things touching .proc/ stuff until https://github.com/tgstation/tgstation/pull/71161 is merged.

This is to avoid merge conflicts, so if you're planning on making a PR that uses it, you should probably wait. Opening the PR is fine but you'll have to update it after.

Notice: All PRs will have to be updated to Master for the work of https://github.com/tgstation/tgstation/pull/71177

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## /tg/station codebase
+## Fulpstation codebase
```
Comment (LemonInTheDark) [inline README.md:1]:
```suggestion
## /fulp/station codebase
```

**Comment by Mothblocks** [comment]
omg me 😁

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-## /tg/station codebase
+## /fulp/station codebase
```
Comment (GoldenAlpharex) [inline README.md:1]: Sorry I think you typo'd...
```suggestion
## /skyrat/station codebase
```

**Comment by Colovorat** [comment]
merge it

File: README.md (hunk 2 above)
Comment (Jolly-66) [inline README.md:1]: Willard I think you forgot ONE crucial detail.
```suggestion
## /tale/station codebase
```

**Comment by Mothblocks** [comment]
Just a note that maps/sprites have been unintentionally frozen...ish.

I added 515 as a required test. This means no PRs other than AnturK's can be merged. After it is merged, you will need to merge from upstream. This is good.

However, map and sprite team do not have admin on the repo, and cannot override the required checks. I don't think it's a big deal but I figured people should know.

File: README.md (hunk 2 above)
Comment (Fikou) [inline README.md:1]:
```suggestion
## /club penguin/station codebase
```
DONT shorten it

Comment (JohnFulpWillard) [reply README.md:1]: this is too long i think we have to

**Comment by Mothblocks** [comment]
I have temporarily disabled the stale workflow until the freeze is over.

https://github.com/tgstation/tgstation/actions/workflows/stale.yml

**Comment by JohnFulpWillard** [comment]
image

**Comment by Mothblocks** [comment]
@JohnFulpWillard I think it's a bad idea to only lift the freeze on `.proc/`

If we want to lift the freeze we should merge the PR now, with a preprocessor check to block 515+ without UNIT_TESTS.

That PR comes WITH a test to block `.proc/` AND the replacement for it

**Comment by Mothblocks** [comment]
CC @AnturK and @optimumtact on above

**Comment by Mothblocks** [comment]
@tgstation/commit-access Please do not merge any code PRs until I create a test for making sure your branch has the 515 commit merged in, should be done very soon

**Comment by Mothblocks** [comment]
Freeze is over, update your masters.

I will be re-enabling the stale workflow in 3 days.

**Pull request closed without merging**